lib/db/dao/category_topic_dao.dart

```diff
@@ -0,0 +1,39 @@
+import 'dart:async';
+
+import 'package:maui/app_database.dart';
+
+import 'package:sqflite/sqflite.dart';
+
+import '../entity/category_topic.dart';
+
+class CategoryTopicDao {
+  Future<CategoryTopic> getTopic(String categoryId,
```

Comment:
Again since we cannot guarantee that there will be single row, this method is not needed